.github/workflows/ansible-test.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        grafana_version: ["8.1.8",  "8.2.7", "8.3.3"]
+        grafana_version: ["8.2.7",  "8.3.7", "8.4.7"]
```

Comment:
Perhaps update the README with the new versions? 
https://github.com/ansible-collections/community.grafana/blob/ac249c3137039168c000d7830cebb80c8ae35f36/README.md?plain=1#L37